### ISSUE TITLE
Study screenreader compatibility

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#2339]
**Is your feature request related to a problem? Please describe.**
F3D is currently completely unable to be connected to a screen reader.
While we don't plan to add support for traditional screen readers like NVDA (Windows only) or JAWS (closed-source) it would be nice to find a way forward that will led to en eventual support of screen readers

**Study notes**
Traditional screenreaders use dedicated API use custom API that should be loaded using a plugin. Implementing anything custom is out of the question, we would need a standard.
 - Jaws: https://www.freedomscientific.com/products/software/jaws/
 - NVDA: https://github.com/nvaccess/nvda/

Freedesktop created a standard for this but this would make our implementation linux specific and compatible with basically only Orca, that's better but not really a solution for most of our users.
 - AT-SPI2: https://www.freedesktop.org/wiki/Accessibility/AT-SPI2/
 - Orca: https://orca.gnome.org/

Usually, such accessibility feature are handled by the UI framework in use, we happen to use ImGui, so what does ImGui support ?

Not much as of now, however there are pointers that shows that this may added in the feature:
 - ImGui is open to adding screen reader support: https://github.com/ocornut/imgui/issues/4122
 - Nice issue explaining the way forward for ImGui: https://github.com/ocornut/imgui/issues/8022
 - The key: accesskit, cross-platform open source toolkit for accessibility: https://accesskit.dev/

**What now?**

So until ImGui properly supports accesskit, there is nothing to do on our end apart, well, contributing to ImGui if we want it bad enough or at least testing the work of other people implementing this.

**Additional context**

**Contribution**
Please tell us if you are able to contribute (Can you develop this new feature?)
Don't worry, you can still post a feature request if you cannot contribute!
